orca-queue-redis/src/main/kotlin/com/netflix/spinnaker/orca/q/redis/RedisQueue.kt

```diff
@@ -62,6 +63,18 @@ class RedisQueue(
   private val hashKey = "$queueName.hash"
   private val hashesKey = "$queueName.hashes"
 
+  companion object {
+    fun convertToMessage(json: String, mapper: ObjectMapper): Message {
+      val messageMap = mapper.readValue<Map<String, Any>>(json)
+
+      return if (messageMap.containsKey("payload")) {
```

Comment:
Point is to try and be backwards compatible when introducing an envelope (provided that envelope contains the original `Message` as a `payload` attribute).

Once the old orca is no longer running, this change can be reverted in favor of using a yet-to-be-defined `Envelope` across the board.